server.js

```diff
@@ -12,9 +12,15 @@ const fs = require('graceful-fs');
 const webpack = require('webpack');
 const webpackDevConfig = require('./webpack.config.js');
 const webpackMiddleware = require('webpack-dev-middleware');
+const { fetchPortOfEntries, fetchCountryOfResidences } = require('utils/enumJsonLists');
 
 const logPath = 'server.js';
 
+(async function initializeEnums() {
```

Comment:
same pedantic point I made on the backend PR :)
portsOfEntry, countriesOfResidence